src/coredump/coredump.c

```diff
@@ -1221,6 +1231,74 @@ static int process_kernel(int argc, char* argv[]) {
         return r;
 }
 
+static int process_backtrace(int argc, char *argv[]) {
+        const char *context[_CONTEXT_MAX];
+        char *t;
+        _cleanup_free_ char *comm = NULL;
+        struct iovec iovec[27];
+        size_t n_iovec = 0, i, n_to_free;
+        uint8_t buf[4096];
+        ssize_t buf_bytes;
+        int r;
+
+        log_debug("Processing backtrace on stdin...");
+
+        if (argc < CONTEXT_COMM + 1) {
+                log_error("Not enough arguments passed (%i, expected %i).", argc - 1, CONTEXT_COMM + 1 - 1);
+                return -EINVAL;
+        }
+
+        context[CONTEXT_PID]       = argv[CONTEXT_PID + 2];
+        context[CONTEXT_UID]       = argv[CONTEXT_UID + 2];
+        context[CONTEXT_GID]       = argv[CONTEXT_GID + 2];
+        context[CONTEXT_SIGNAL]    = argv[CONTEXT_SIGNAL + 2];
+        context[CONTEXT_TIMESTAMP] = argv[CONTEXT_TIMESTAMP + 2];
+        context[CONTEXT_RLIMIT]    = argv[CONTEXT_RLIMIT + 2];
+
+        r = gather_pid_metadata(context, argv + CONTEXT_COMM + 2, &comm, iovec, &n_iovec);
+        if (r < 0)
+                goto finish;
+
+        if (isempty(context[CONTEXT_SIGNAL]))
+                context[CONTEXT_SIGNAL] = "an exception";
```

Comment:
This part is removed in one of the later commits. I'd leave it as is because of that.